tests/parser/EclipseGridTests.cpp

```diff
@@ -2148,6 +2148,64 @@ BOOST_AUTO_TEST_CASE(TESTCP_ACTNUM_UPDATE) {
 }
 
 
+BOOST_AUTO_TEST_CASE(TESTCP_ACTNUM_AQUNUM) {
+    const char* deckData =
```

Comment:
Please use multiline string literals:
```C++
const std::string deckData = R"(
RUNSPEC

DIMENS
   3 2 1 

GRID

COORD
...
)";
```
instead of `const char* ` - then you can void all the `"` and `\n`

Reply:
Sure. I just copied the case from another test in the same file, so it was not a big problem. But I will change to the raw string format.